lib/jekyll/commands/serve.rb

```diff
@@ -13,9 +15,22 @@ class << self
           "show_dir_listing"   => ["--show-dir-listing",
             "Show a directory listing instead of loading your index file."],
           "skip_initial_build" => ["skip_initial_build", "--skip-initial-build",
-            "Skips the initial site build which occurs before the server is started."]
+            "Skips the initial site build which occurs before the server is started."],
+          "livereload"         => ["-l", "--livereload",
+            "Use LiveReload to automatically refresh browsers"],
+          "swf"                => ["--swf", "Use Flash for WebSockets support"],
+          "ignore"             => ["--ignore GLOB1[,GLOB2[,GLOB3...]]", Array,
```

Comment:
If these options belong to live-reload directly and not to Jekyll globally then please prefix them with `--live-reload-`